deployer/templates/fluentd.yaml

```diff
@@ -329,13 +323,9 @@ parameters:
   name: USE_MUX
   value: "false"
 -
-  description: 'Configure MUX CLIENT (false|true).'
-  name: USE_MUX_CLIENT
-  value: "false"
--
-  description: 'Configure MUX SERVER (false|true).'
-  name: MUX_ALLOW_EXTERNAL
-  value: "false"
+  description: 'Configure MUX CLIENT MODE (minimal|full_no_k8s_meta).'
```

Comment:
If we replace the fluentd collector in the future with something else, would this option still be applicable?  Do you believe this name is general enough that it would apply in other situations?  If you do, then let's keep it.  If you don't, would not a more general name be appropriate for use where we can keep the name but define the behavior appropriately via documentation, so that user's don't have to change their inventory files in ansible, or scripts?